hypertrace-trace-enricher/hypertrace-trace-enricher/src/main/resources/configs/common/application.conf

```diff
@@ -67,7 +67,7 @@ enricher {
       port = 50061
       port = ${?ENTITY_SERVICE_PORT_CONFIG}
     }
-    dependencies = ["DefaultServiceEntityEnricher"]
+    dependencies = ["DefaultServiceEntityEnricher", "EndpointEnricher"]
```

Comment:
I wonder why do we have all this dependency information as part of config. Do we need to frequently change this order based on environment etc?

Reply:
I don't think we change this based on the environment but we want to be able to enable/disable a given enricher.